### PR TITLE
this is test class and this is my 1st Git program

### PR DESCRIPTION
**Pull request by rajupalumaru**
this is test class and this is my 1st Git program

### CODE REVIEW

File: src/main/java/com/test/Test.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.test;
+
+public class Test {
+	public static void main(String[] args) {
+		System.out.println("test class");
```
Comment (rajupalumaru) [inline src/main/java/com/test/Test.java]: change the code and not mandatiry in sysout

File: src/main/java/com/test/Test.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package com.test;
+
+public class Test {
+	public static void main(String[] args) {
+		System.out.println("test class");
+		System.out.println("once check it ");
```
Comment (rajupalumaru) [inline src/main/java/com/test/Test.java]: delete one syso

Comment (nareshreddy12) [inline src/main/java/com/test/Test.java]: insted of main method create your own method

Comment (yadavenkatesh) [reply src/main/java/com/test/Test.java]: ok

Comment (yadavenkatesh) [reply src/main/java/com/test/Test.java]: ok

**Pull request merged**